adapters/adnuntius/adnuntius.go

```
@@ -344,6 +350,18 @@ func generateAdResponse(ad Ad, impId string, html string, request *openrtb2.BidR
 		}}
 	}
 
+	price := ad.Bid.Amount
+	priceType, _, _, _ := jsonparser.Get(imp.Ext, "bidder", "priceType")
```

Please do not ignore errors.